thirdparty/misc/clipper.cpp

```diff
@@ -4329,10 +4336,10 @@ double DistanceFromLineSqrd(
   const IntPoint& pt, const IntPoint& ln1, const IntPoint& ln2)
 {
   //The equation of a line in general form (Ax + By + C = 0)
-  //given 2 points (x�,y�) & (x�,y�) is ...
-  //(y� - y�)x + (x� - x�)y + (y� - y�)x� - (x� - x�)y� = 0
-  //A = (y� - y�); B = (x� - x�); C = (y� - y�)x� - (x� - x�)y�
-  //perpendicular distance of point (x�,y�) = (Ax� + By� + C)/Sqrt(A� + B�)
+  //given 2 points (x¹,y¹) & (x²,y²) is ...
```

Comment:
Encoding change?

Reply:
This is weird one because `git diff` makes encoding look like this, but the encoding could still be preserved while committing changes... so I left it this way.

Setting `LESSCHARSET=UTF-8 git diff` doesn't seem to cure this either.

Reply:
`git add -p` should let you selectively add changes and thus ignore those if they're generated by your OS and you can't find how to fix them (I've had similar issues in the past too with weird Mac line endings).

Reply:
Used `git add -p`, `VSCode` seems to mess up encoding. 🐛

Reply:
Talking about tools bug, it's weird that your `.patch` file appears as a binary blob in the GitHub diff. In my experience GitHub used to recognize those as text files, but maybe only when created on Unix? See e.g. https://github.com/godotengine/godot/pull/27929/commits/668439d16a5c30587ab77461fb3211306ad62587

Reply:
I checked the encoding for those, and the patch in 668439d appears as UTF-8 LF, while the patch generated in Windows Git generates UTF-16LE CRLF, but maybe because it's created locally and wasn't checked out from remote yet.

Reply:
#29098 should fix the encoding issue with these comments, you might have to fight git a bit when rebasing, but hopefully it should be fine since your PR no longer changes this code chunk.

I also pushed a change to enforce LF line endings in the git repo (converted automatically to CRLF when checking out on Windows, and converted back to LF when committing) for all text files, including `.patch` files, so this might solve the diff issue.

Reply:
Yeah there's plenty of changes so far, I'm going to rebase once it's merged.